examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -78,14 +79,17 @@ function parseTilesetUrlFromUrl() {
 
 function parseTilesetUrlParams(url, options) {
   const parsedUrl = new URL(url);
-  const index = url.lastIndexOf('/layers/0');
-  let metadataUrl = url.substring(0, index);
   let token = options && options.token;
+  const tilesetUrl = createTilesetUrl(parsedUrl);
```

Comment:
I'd probably move this entire method into the helper library.

Reply:
Moved